src/code/z_actor_dlftbls.c

```diff
@@ -24,20 +24,26 @@
 
 // Actor Overlay Table definition
 #define DEFINE_ACTOR(name, _1, allocType, nameString) \
-    { (uintptr_t)_ovl_##name##SegmentRomStart,        \
-      (uintptr_t)_ovl_##name##SegmentRomEnd,          \
-      _ovl_##name##SegmentStart,                      \
-      _ovl_##name##SegmentEnd,                        \
-      NULL,                                           \
-      &name##_InitVars,                               \
-      nameString,                                     \
-      allocType,                                      \
-      0 },
-
-#define DEFINE_ACTOR_INTERNAL(name, _1, allocType, nameString) \
-    { 0, 0, NULL, NULL, NULL, &name##_InitVars, nameString, allocType, 0 },
-
-#define DEFINE_ACTOR_UNSET(_0) { 0 },
+    {                                                 \
+        ROM_FILE(ovl_##name),                         \
+        _ovl_##name##SegmentStart,                    \
+        _ovl_##name##SegmentEnd,                      \
+        NULL,                                         \
+        &name##_InitVars,                             \
+        nameString,                                   \
+        allocType,                                    \
+        0,                                            \
+    },
+
+#define DEFINE_ACTOR_INTERNAL(name, _1, allocType, nameString)                        \
+    {                                                                                 \
+        ROM_FILE_UNSET, NULL, NULL, NULL, &name##_InitVars, nameString, allocType, 0, \
+    },
+
+#define DEFINE_ACTOR_UNSET(_0)                              \
+    {                                                       \
+        ROM_FILE_UNSET, NULL, NULL, NULL, NULL, '\0', 0, 0, \
```

Comment:
```suggestion
        ROM_FILE_UNSET, NULL, NULL, NULL, NULL, NULL, 0, 0, \
```
but I'd just keep the existing 0 initializer `{0}` tbh